### PR TITLE
Update-ndc-spec-0.1.3

### PR DESCRIPTION
**Pull request by BenoitRanque**
Update to ndc spec 0.1.3

Note: also updating the sdk version to the same number, is that a good idea?

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [workspace]
 resolver = "2"
 
-package.version = "0.1.0"
+package.version = "0.1.3"
```
Comment (BenoitRanque) [inline Cargo.toml:4]: Bumped version from 0.1.0 to 0.1.3, to keep the same version number as the corresponding ndc-spec.

Is this a good idea, should we do something else instead? The thinking is having a single number wil be less confusing.

File: crates/sdk/src/connector/example.rs [hunk 2]
```diff
@@ -50,13 +50,17 @@ impl Connector for Example {
 
     async fn get_capabilities() -> JsonResponse<models::CapabilitiesResponse> {
         models::CapabilitiesResponse {
-            version: "0.1.1".into(),
+            version: "0.1.3".into(),
```
Comment (BenoitRanque) [inline crates/sdk/src/connector/example.rs:53]: Very unsure about this, I _think_ version is supposed to be the ndc-spec version, is that correct?

File: crates/sdk/src/default_main.rs [hunk 3]
```diff
@@ -84,6 +84,8 @@ struct TestCommand {
     configuration: PathBuf,
     #[arg(long, value_name = "DIRECTORY", env = "HASURA_SNAPSHOTS_DIR")]
     snapshots_dir: Option<PathBuf>,
+    #[arg(long, help = "Turn off validations for query responses")]
+    no_validate_responses: bool,
```
Comment (BenoitRanque) [inline crates/sdk/src/default_main.rs:88]: this flag copied from the [ndc-test implementation](https://github.com/hasura/ndc-spec/pull/141/commits/b6f01f54ee254e277effcfffa42817660c1f7251)

**Comment by soupi** [comment]
fyi this is the third PR implementing this #11 #12

**Review by soupi — APPROVED** [review]

**Comment by BenoitRanque** [comment]
> fyi this is the third PR implementing this #11 #12

😂 Oops! ~~We can just merge @paf31 's PR then?~~

**Pull request merged**